src/pages/projects/introduction-to-cloudflare-workers/index.tsx

```diff
@@ -0,0 +1,342 @@
+      },
+      {
+        title:
+          'Deploy to a Custom Domain with Cloudflare Wrangler Environments',
+        slug:
+          'cloudflare-deploy-to-a-custom-domain-with-cloudflare-wrangler-environments',
+        path:
+          '/lessons/cloudflare-deploy-to-a-custom-domain-with-cloudflare-wrangler-environments',
+      },
+    ],
+  }
+
+  return (
+    <>
+      <div className="mb-10 pb-10 xl:px-0 px-5 max-w-screen-xl mx-auto">
+        <div className="mt-10 mb-16 text-center">
+          <div className="mb-10">
+            <Image priority src={course.image} height="270" width="270" />
+          </div>
+          <p className="text-lg md:text-2xl leading-6 text-gray-500">
+            Portfolio Project Challenge
+          </p>
+          <h1 className="text-2xl md:text-4xl font-bold mt-2">
+            {course.title}
+          </h1>
+        </div>
+        <ProjectBrief
+          className="pb-12"
+          topic={{
+            name: 'cloudflare',
+            label: 'Project Brief',
+          }}
+        >
+          <Markdown className="prose dark:prose-dark pt-2 sm:text-base text-sm leading-normal text-gray-800 dark:text-gray-200 mt-0">
+            {`You are a developer for a national concert promoter.
+
+Music fans from all over the world visit your site to see when and where the next show is going to be. To save them time when they visit the page, we want to show them upcoming events at a venue close to their location.
+
+Your task is to create and deploy a Cloudflare Worker that will examine the request for location data, and render HTML featuring information for the closest concert taking place based on the nearest [regional Cloudflare Location](https://www.cloudflare.com/network/). Choose locations for testing where you have friends (or a VPN 😅) so you can prove this functionality works as expected.`}
+          </Markdown>
+        </ProjectBrief>
+
+        <div className="flex justify-center">
+          <svg
+            width="7"
+            height="155"
+            viewBox="0 0 7 155"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <line
+              x1="3.5"
+              y1="-1.5299e-07"
+              x2="3.50001"
+              y2="155"
+              stroke="#B0B0B0"
+              strokeWidth="7"
+              strokeDasharray="10 10"
+            />
+          </svg>
+        </div>
+
+        <div className="grid grid-cols-1 md:grid-cols-3 mt-16 mb-16">
+          <div className="relative px-10 py-10 bg-white dark:border-gray-800 dark:bg-gray-800 dark:text-gray-200 col-span-2 shadow rounded-md border border-gray-100 sm:mr-0 md:mr-4">
+            <div
+              className="absolute rounded-t-lg rounded-b-none left-0 top-4 -mt-5 h-3 w-full bg-gradient-to-r"
+              style={{
+                background:
+                  'linear-gradient(to right, #F5C361 0%, #E75E3C 100%)',
```

Comment:
figma makes generating these gradients easy though you have to tweak the styles when using them in a React component